### PR TITLE
Break out predictors

### PR DESCRIPTION
**Pull request by bfolie**
# Citrine Python PR

## Description 
One file per predictor

### PR Type:
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

### Adherence to team decisions
- [ ] I have added tests for 100% coverage
- [ ] I have written Numpy-style docstrings for every method and class.
- [ ] I have communicated the downstream consequences of the PR to others.
- [x] I have bumped the version in setup.py

### CODE REVIEW

File: src/citrine/informatics/predictors/expression_predictor.py [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+
+    def _post_dump(self, data: dict) -> dict:
+        data['display_name'] = data['config']['name']
+        return data
+
+    def __str__(self):
+        return '<ExpressionPredictor {!r}>'.format(self.name)
+
+
+class DeprecatedExpressionPredictor(Serializable['DeprecatedExpressionPredictor'], Predictor):
```
Comment (bfolie) [inline src/citrine/informatics/predictors/expression_predictor.py:73]: @latture how do we decide when we can remove this?

File: src/citrine/informatics/predictors/__init__.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+from .predictor import *
+from .expression_predictor import *
+from .simple_ml_predictor import *
+from .generalized_mean_property_predictor import *
+from .graph_predictor import *
+from .ingredient_fractions_predictor import *
+from .ingredients_to_simple_mixture_predictor import *
+from .label_fractions_predictor import *
+from .molecular_structure_featurizer import *
+from .simple_mixture_predictor import SimpleMixturePredictor
```
Comment (talltea) [inline src/citrine/informatics/predictors/__init__.py:11]: why import these explicitly and not with `*`? (not blocking, just curious)

**Review by talltea — DISMISSED** [review]
👍

File: src/citrine/informatics/predictors/expression_predictor.py (hunk 1 above)
Comment (latture) [reply src/citrine/informatics/predictors/expression_predictor.py:73]: We should follow up with product/DS to see if it's still being used. My guess is that it's probably safe, but we should check and potentially make an announce that it's being removed in the next version first.

File: src/citrine/informatics/predictors/__init__.py (hunk 2 above)
Comment (bfolie) [reply src/citrine/informatics/predictors/__init__.py]: That's an oversight. Fixed

**Comment by latture** [comment]
@bfolie One thing I've noticed by breaking out the other modules into separate files is that this breaks class references in our user docs. We want to check if that's the case here, but for workflows we had to use the full path, e.g. change :class:`~citrine.informatics.workflows.DesignWorkflow` to :class`~citrine.informatics.workflows.design_workflow.DesignWorkflow`

**Comment by bfolie** [comment]
> @bfolie One thing I've noticed by breaking out the other modules into separate files is that this breaks class references in our user docs. We want to check if that's the case here, but for workflows we had to use the full path, e.g. change :class:`~citrine.informatics.workflows.DesignWorkflow` to :class`~citrine.informatics.workflows.design_workflow.DesignWorkflow`

Good catch, thanks. I'll check and update the docs.

**Review by talltea — APPROVED** [review]

**Pull request merged**